### PR TITLE
ZK-4719: tablet-ui disabled applies only partially

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zk/src/org/zkoss/zk/ui/http/Wpds.java [hunk 1]
```diff
@@ -145,6 +145,18 @@ public static final String outMomentTimezoneJavascript(ServletRequest request, S
 		return result.toString();
 	}
 
+	/**
+	 * Internal use only.
+	 * Generates some Library-properties strings in JavaScript syntax
+	 * @since 9.5.1
+	 */
+	public static final String outLibraryPropertyJavaScript() {
+		StringBuffer sb = new StringBuffer();
```
Comment (plijyfes) [inline zk/src/org/zkoss/zk/ui/http/Wpds.java:154]: builder

File: zk/src/archive/web/js/zk/zk.wpd [hunk 2]
```diff
@@ -51,7 +53,10 @@ window.$eval=function(s){return eval(s);};
 	<script src="ext/fetch.js"/>
 	<script src="au.js" jsdoc="true"/>
 	<script src="flex.js"/>
-	<script>if (zk.mobile) {</script>
+	<script><![CDATA[
+		var tabletSupported = zk.mobile && !zk.tabletUIDisabled;
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/zk.wpd:57]: zk.touchEnabled

File: zktest/src/archive/test2/B95-ZK-4719.zul [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+				Mon Nov 16 12:52:35 CST 2020, Created by leon
+
+Copyright (C) 2020 Potix Corporation. All Rights Reserved.
+
+-->
+<zk>
+	<zscript>
+		org.zkoss.lang.Library.setProperty("org.zkoss.zkmax.tablet.ui.disabled", "true");
+	</zscript>
+	<label multiline="true">
```
Comment (plijyfes) [inline zktest/src/archive/test2/B95-ZK-4719.zul:19]: add a button to reset the lib property after test

**Review by scribetw — CHANGES_REQUESTED** [review]
should refine

**Review by scribetw — APPROVED** [review]

File: zk/src/archive/web/js/zk/zk.wpd [hunk 4]
```diff
@@ -28,6 +28,8 @@ window.$eval=function(s){return eval(s);};
 	<script src="zk.js" jsdoc="true"/>
```
Comment (scribetw) [inline zk/src/archive/web/js/zk/zk.wpd:28]: add macintel, tabletUIDisabled, and touchEnabled jsdoc

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by DevChu — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**